lib/simple_form/components/maxlength.rb

```diff
@@ -29,6 +26,20 @@ def find_length_validator
       def has_tokenizer?(length_validator)
         length_validator.options[:tokenizer]
       end
+
+      # Use validation with tokenizer if version of Rails is less than 5,
+      # if not validate without the tokenizer, if version is greater than Rails 4.
+      def validation_with_tokenizer(length_validator)
+        if ActionPack::VERSION::STRING < '5'
```

Comment:
Put the conditional outside the method to avoid runtime conditionals when we know the value in load time.